### PR TITLE
Capistrano

### LINKED ISSUE #20: Capistrano
Once we decide where we're going to deploy the app, someone needs to set up Capistrano to match.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #20 

I also did some rubocop stuff.

### CODE REVIEW

File: config/deploy/production.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+remote_user = Net::SSH::Config.for('af-transit-app3.admin.umass.edu')[:user] || ENV['USER']
```
Comment (dfaulken) [inline config/deploy/production.rb]: This will change, but we can merge for now.

File: config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+set :repo_url, 'git@github.com:umts/st-pax.git'
+set :branch, :master
+
+set :keep_releases, 5
+
+set :deploy_to, "/srv/#{fetch :application}"
+
+set :log_level, :info
+
+set :whenever_command, %i[sudo bundle exec whenever]
```
Comment (dfaulken) [inline config/deploy.rb]: We should make sure that `PATH` is set correctly in the `config/schedule.rb` file.

See https://github.com/umts/jobapps/pull/300.

Comment (mboneil10) [reply config/deploy.rb]: I don't. What.

Comment (mboneil10) [reply config/deploy.rb]: Literally, just that line?

Comment (dfaulken) [reply config/deploy.rb]: Yes, we should add that line.

**Comment by dfaulken** [comment]
@mboneil10 I merged #51 to remove some of the noise from this PR.

**Pull request merged**